docs/source/release_notes.rst

```diff
@@ -0,0 +1,213 @@
+.. SPDX-FileCopyrightText: ASSUME Developers
+..
+.. SPDX-License-Identifier: AGPL-3.0-or-later
+
+#######################
+Release Notes
+#######################
+
+Upcoming Release 0.3.8
+=======================
+
+**New Features:**
+
+- **Market Coupling:** Users can now perform market clearing for different market zones with given transmission capacities. This feature
+  allows for more realistic simulation of market conditions across multiple interconnected regions, enhancing the accuracy of market
+  analysis and decision-making processes. A tutorial on how to use this feature is coming soon.
+
+- **Adjust the Framework to Schedule Storing to the Learning Role:** This enhancement enables Learning agents to participate in sequential
+  markets, such as day-ahead and intraday markets. The rewards are now written after the last market, ensuring that the learning process
+  accurately reflects the outcomes of all market interactions. This improvement supports more sophisticated and realistic agent training scenarios.
+  A tutorial on how to use this feature is coming soon.
+
+**Improvements:**
+
+- Significant speed up of the framework and especially of the learning process
+- Split scenario loader function to improve speed and reduce unrequired operations
+- Refactored unit operator by adding a seperate unit operator for learning units
```

Comment:
separate

Reply:
Sorry, I meant that there is a typo: `separate` instead of `seperate :D
though your change is also fine :D